src/main/scala/dpla/ingestion3/harvesters/file/LcCsvFileHarvester.scala

```diff
@@ -0,0 +1,152 @@
+
+import java.io.File
+
+import com.databricks.spark.avro._
+import dpla.ingestion3.confs.i3Conf
+import dpla.ingestion3.mappers.utils.JsonExtractor
+import org.apache.log4j.Logger
+import org.apache.spark.sql.functions._
+import org.apache.spark.sql.{DataFrame, SparkSession}
+import org.json4s.jackson.JsonMethods._
+import org.json4s.{JValue, _}
+
+import scala.io.Source
+import scala.util.{Failure, Success, Try}
+
+
+/**
+  * Extracts values from parsed JSON
+  */
+class CsvFileExtractor extends JsonExtractor
+
+/**
+  * Entry for performing a CSV file harvest
+  */
+class LcCsvFileHarvester(spark: SparkSession,
+                         shortName: String,
+                         conf: i3Conf,
+                         logger: Logger)
+  extends FileHarvester(spark, shortName, conf, logger) {
+
+  def mimeType: String = "application_json"
+
+  protected val extractor = new CsvFileExtractor()
+
+  /**
+    * Loads .csv files
+    *
+    * @param file File to parse
+    * @return Iterator[String] of the CSV contents
+    */
+  def getInputStream(file: File): Option[Iterator[String]] = {
+    file.getName match {
+      case csvName if csvName.endsWith("csv") => // FIXME isn't the redundant of the file filter? 
+        Some(Source.fromFile(file).getLines())
+      case _ => None
+    }
+  }
+
+  /**
+    * Parses JValue to extract item local item id and renders compact
+    * full record
+    *
+    * @param json Full JSON item record
+    * @return Option[ParsedResult]
+    */
+  def getJsonResult(json: JValue): Option[ParsedResult] =
+    Option(ParsedResult(
+      extractor.extractString(json \\ "id")
+        .getOrElse(throw new RuntimeException("Missing ID")),
+      compact(render(json))
+    ))
+
+  /**
+    * Parses and extracts ZipInputStream and writes
+    * parsed records out.
+    *
+    * @param row  Case class representing extracted items from a CSV row
+    * @return Count of metadata items found.
+    */
+  def handleRow(row: CsvRow,
+                 unixEpoch: Long): Try[Int] = {
+    row match {
+      case CsvRow(_, None) =>
+        Success(0) // a directory, no results
+      case CsvRow(Some(_), Some(data)) => Try {
+        Try {
+
+          // Clean up leading/trailing characters
+          val json: JValue = parse(
+            data
+            .stripPrefix("\"") // remove leading double quote
+            .stripSuffix("\"") // remove trailing double quote
+            .stripPrefix("[")
+            .stripPrefix(",")
+            .replaceAll("\"\"", "\"") // replace double-double quote with single double quote
+          )
+
+          getJsonResult(json) match {
+            case Some(item) =>
+              writeOut(unixEpoch, item)
+              1
+            case _ => 0
+          }
+        } match {
+          case Success(num) => num
+          case _ => 0
+        }
+      }
+    }
+  }
+
+  override def handleFile(fileResult: FileResult, unixEpoch: Long): Try[Int] = ???
```

Comment:
@mdellabitta `handleCsv` replaces `handleFile` but I had to stub this method to extend `FileHarvster`.  I may have overlooked a way of doing this that conforms to what `handleFile` expects.